webapp/src/js/selectors/index.js

```diff
@@ -9,6 +11,29 @@ angular.module('inboxServices').factory('Selectors', function () {
   var getSelectMode = function(state) { return state.selectMode; };
   var getSelected = function(state) { return state.selected; };
 
+  var getSelectedSummaries = reselect.createSelector(
+    getSelected,
+    function(selected) {
+      if (!Array.isArray(selected)) {
+        return [];
+      }
+      return selected.map(function(item) {
```

Comment:
Have you considered writing this code in es6 notation?  For example `selected.map(item => item.formatted || item.summary)`? I'm excited to see the modern redux patterns coming into our webapp, but it would be great if the work brought a modern syntax as well.

Reply:
Hi Kenn, I would love to be writing all of this in es6 notation! I was under the impression webapp code still needed to be compatible with an older version of chrome and thus using es5, has that changed?

Reply:
Yessir! It has. WebApp 3.x [requires the latest container with Chrome 53](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md) - which supports es6.  Browserify and uglify don't support everything, and you [can't set breakpoints inside of lambdas](https://github.com/medic/medic/issues/5206), but a lot of features work.

Reply:
Oh my good gosh, thanks for letting me know I'll update these then 👍

Reply:
Oh my good gosh, thanks for letting me know I'll update these then 👍

Reply:
Does this mean crosswalk was able to be updated for newer versions of Chrome? Or is it no longer being used?